### PR TITLE
HyperShift: Move CNCC to the controll-plane namespace

### PR DESCRIPTION
**Pull request by kyrtapz**
When running in HyperShift deploy CNCC in the management cluster.

Signed-off-by: Patryk Diak <pdiak@redhat.com>

### CODE REVIEW

File: pkg/network/cloud_network.go [hunk 1]
```diff
@@ -1,6 +1,8 @@
 package network
 
 import (
+	"github.com/openshift/cluster-network-operator/pkg/client"
```
Comment (alvaroaleman) [inline pkg/network/cloud_network.go:4]: Tiny nit: Maybe use a different import group?

**Review by alvaroaleman — COMMENTED** [review]
The change looks good, I didn't actually test it out though

File: pkg/network/cloud_network.go [hunk 2]
```diff
@@ -38,18 +41,27 @@ func renderCloudNetworkConfigController(conf *operv1.NetworkSpec, cloudBootstrap
 	data.Data["PlatformTypeAzure"] = v1.AzurePlatformType
 	data.Data["PlatformTypeGCP"] = v1.GCPPlatformType
 	data.Data["CloudNetworkConfigControllerImage"] = os.Getenv("CLOUD_NETWORK_CONFIG_CONTROLLER_IMAGE")
-	data.Data["KubernetesServiceHost"] = cloudBootstrapResult.APIServers[bootstrap.APIServerDefault].Host
-	data.Data["KubernetesServicePort"] = cloudBootstrapResult.APIServers[bootstrap.APIServerDefault].Port
+	data.Data["KubernetesServiceHost"] = cloudBootstrapResult.APIServers[bootstrap.APIServerDefaultLocal].Host
```
Comment (zshi-redhat) [inline pkg/network/cloud_network.go:44]: @kyrtapz  I forgot the difference between APIServerDefaultLocal and APIServerDefault, do we need to distinguish it for self-hosted and managed?

Comment (kyrtapz) [reply pkg/network/cloud_network.go:44]: this variable should do exactly that: https://github.com/openshift/cluster-network-operator/blob/3b3909bd4f64390fc2b0c80b1be92ff6370209a5/pkg/bootstrap/types.go#L105-L108

Comment (kyrtapz) [reply pkg/network/cloud_network.go:44]: Yes, that is correct

**Pull request merged**